pkg/sql/logical_schema_accessors.go

```diff
@@ -87,26 +88,35 @@ func (l *LogicalSchemaAccessor) GetObjectDesc(
 ) (catalog.ObjectDescriptor, error) {
 	switch flags.DesiredObjectKind {
 	case tree.TypeObject:
+		// TODO(ajwerner): Change this function if we ever expose non-table objects
+		// underneath virtual schemas. For now we assume that the only objects
+		// ever handed back from GetObjectByName is
```

Comment:
sentence is not complete (commit "sql: adopt VirtualSchemas ...")

Reply:
I fixed up the comment and also the logic, it didn't make sense to construct an `UncachedPhysicalAccessor` here, rather I called through to the `CachedPhysicalAccessor` and had that call through. It should make testing easier when we get around to unit testing this stuff.

Reply:
Thanks.